google/cloud/security/notifier/notifier.py

```diff
@@ -221,6 +223,11 @@ def main(_):
     for pipeline in pipelines:
         pipeline.run()
 
+    if notifier_configs.get('violation').get('cscc').get('enabled'):
```

Comment:
Do you think it's possible that the user only update the application and didn't update the configuration? This might throw null pointer exception.

Reply:
Good catch!  Done.